src/object_store.jl

```diff
@@ -25,11 +25,10 @@ captured exception will be thrown on `get`.
 get(obj_ref::ObjectRef) = get(obj_ref.oid)
 
 function get(oid::ray_jll.ObjectIDAllocated)
-    local ray_obj
-    while true
-        ray_obj = ray_jll.get(oid, 0)
-        isnull(ray_obj[]) ? sleep(0.1) : break
+    while !ray_jll.contains(oid)
+        sleep(0.1)
     end
```

Comment:
can this now be refactored to use `Base.wait`?

Reply:
ah yes, good catch.

Reply:
ah wait, maybe not.  unless we define `Base.wait` on `ObjectID`